R/rABS.R

```diff
@@ -0,0 +1,94 @@
+#' ABS simulation
+#'
+#' This function is for simulating the Auto-correlated Bayesian Sampler \insertCite{zhuAutocorrelatedBayesian2023}{samplr}.
```

Comment:
This could be expanded -- we may want to add more detail here or write a vignette on how ABS works (your choice)

Reply:
I think we may need to write a vignette or add a section in the ABS parameter recovery one.

Reply:
Great -- I've added it to an issue (#4)